lib/rspec/matchers/built_in/eq.rb

```diff
@@ -20,7 +20,8 @@ def failure_message_when_negated
         # @api private
         # @return [String]
         def description
-          "#{name_to_sentence} #{@expected.inspect}"
+          english_name = EnglishPhrases.humanize(name)
+          "%s %s" % [english_name, @expected.inspect]
```

Comment:
I much prefer using string interpolation to using format strings and `%` as you've done here and below.  Among other things, this form does an extra array allocation.